woocommerce/woocommerce-ios
# [Jetpack setup] UI updates for account creation support

Closes: #14421 


## Description
This PR just adds some UI changes for better support of the account creation flow:
- Adds a note to the email screen about account creation.
- Updates the error message if the user uses an unknown username to indicate that they should enter an email instead.
- Adds a note to the magic link screen about confirming the correct email.

## Steps to reproduce
1. Sign in using Application Passwords.
2. Open the Jetpack setup flow from the dashboard.

## Testing information
- Confirm the presence of the new note on the email screen.
- Enter an unknown username and confirm the new error message is shown.
- Start magic link flow (either signup or login) and confirm that the screen has the new note.

## Screenshots
| Email screen | Username error | Magic Link screen |
| --- | --- | --- |
| ![Simulator Screenshot - iPhone 16 - 2024-11-19 at 17 05 20](https://github.com/user-attachments/assets/54a89f1a-5b23-4ed1-a31a-987ca0ec4ad0) | ![Simulator Screenshot - iPhone 16 - 2024-11-19 at 17 05 33](https://github.com/user-attachments/assets/ea06ec2a-ff38-4dc0-a4d1-3521a2232eb4) |  ![Simulator Screenshot - iPhone 16 - 2024-11-19 at 17 05 52](https://github.com/user-attachments/assets/703c84bf-fef5-483b-9fd8-20fd5c3f804f) |

---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
